### PR TITLE
Favicon support

### PR DESCRIPTION
**Pull request by buzztaiki**
Hi,
I have create [Favicon Plugin](https://github.com/buzztaiki/jenkins-favicon-plugin).
When I sent [Hosting Request](https://issues.jenkins-ci.org/browse/HOSTING-44) to Jenkins JIRA, they suggest that integrate it into Simple Theme Plugin.
This PR adds favicon support into this plugin.

### CODE REVIEW

**Comment by batmat** [comment]
Ping @jenkinsci/code-reviewers would be great to review.
That PR may be an sensible alternative for users to the creation of a brand new plugin _only_ for the favicon addition.

(Cf. [HOSTING-44](https://issues.jenkins-ci.org/projects/HOSTING/issues/HOSTING-44) if need be)

**Comment by batmat** [comment]
Ping @buzztaiki 

This plugin is actually to be adopted, as you wanted to have your favicon plugin forked, I suppose you'd be interested to become the maintainer of the simple-theme-plugin right?

File: src/main/java/org/codefirst/SimpleThemeDecorator.java [hunk 1]
```diff
@@ -21,6 +22,7 @@ public boolean configure(StaplerRequest req, JSONObject formData)
 			throws FormException {
 		cssUrl = formData.getString("cssUrl");
 		jsUrl = formData.getString("jsUrl");
+		faviconUrl = formData.getString("faviconUrl");
```
Comment (batmat) [inline src/main/java/org/codefirst/SimpleThemeDecorator.java]: Btw @buzztaiki you could use bindJson instead to be less verbose here.

**Comment by TobiX** [comment]
Merged manually.

**Pull request closed without merging**